mss/dextre/dextre.go

```diff
@@ -0,0 +1,227 @@
+	"orvedb.aupads.org",
+	// "csi.cloudmark.com",
+	"b.barracudacentral.org",
+	// "sbl-xbl.spamhaus.org",
+	"spam.dnsbl.sorbs.net",
+	// "blackholes.mail-abuse.org",
+	"virus.rbl.jp",
+	// "bl.csma.biz",
+	// "unconfirmed.dsbl.org",
+	// "combined.njabl.org",
+	"relays.bl.kundenserver.de",
+	// "l2.bbfh.ext.sorbs.net",
+	"zombie.dnsbl.sorbs.net",
+	"ubl.unsubscore.com",
+	// "wingate.opm.blitzed.org",
+	// "dialups.visi.com",
+	"netblock.pedantic.org",
+	// "bl.score.senderscore.com",
+	"korea.services.net",
+	// "virbl.bit.nl",
+	// "dialups.mail-abuse.org",
+	// "l2.spews.dnsbl.sorbs.net",
+	"misc.dnsbl.sorbs.net",
+	"virus.rbl.msrbl.net",
+	// "multihop.dsbl.org",
+	// "dialup.blacklist.jippg.org",
+	"dnsbl.sorbs.net",
+	// "rmst.dnsbl.net.au",
+	// "omrs.dnsbl.net.au",
+	// "ucepn.dnsbl.net.au",
+	// "owps.dnsbl.net.au",
+	"zen.spamhaus.org",
+	// "fl.chickenboner.biz",
+	"smtp.dnsbl.sorbs.net",
+	// "spam.olsentech.net",
+	// "spamtrap.trblspam.com",
+	// "bl.technovision.dk",
+	"bl.emailbasura.org",
+	// "proxy.bl.gweep.ca",
+	// "all.spamrats.com",
+	"spam.spamrats.com",
+	"wormrbl.imp.ch",
+	// "work.drbl.gremlin.ru",
+	// "intruders.docs.uu.se",
+	// "relays.mail-abuse.org",
+	"dul.dnsbl.sorbs.net",
+	// "rdts.dnsbl.net.au",
+	"cbl.abuseat.org",
+	"cdl.anti-spam.org.cn",
+	// "query.senderbase.org",
+	// "rbl2.triumf.ca",
+	// "orid.dnsbl.net.au",
+	// "multi.surbl.org",
+	"dnsbl.justspam.org",
+	"combined.rbl.msrbl.net",
+	"virbl.dnsbl.bit.nl",
+	// "spam.wytnij.to",
+	"tor.dnsbl.sectoor.de",
+	"pbl.spamhaus.org",
+	// "whois.rfc-ignorant.org",
+	// "osrs.dnsbl.net.au",
+	"bl.spameatingmonkey.net",
+	"cidr.bl.mcafee.com",
+	// "dul.ru",
+	// "no-more-funn.moensted.dk",
+	"dnsbl-1.uceprotect.net",
+	"db.wpbl.info",
+	// "rbl.suresupport.com",
+	// "dsbl.dnsbl.net.au",
+	// "sbl.csma.biz",
+	"phishing.rbl.msrbl.net",
+	"rbl.dns-servicios.com",
+	// "spews.dnsbl.net.au",
+	// "access.redhawk.org",
+	"spamrbl.imp.ch",
+	// "proxy.block.transip.nl",
+	// "torserver.tor.dnsbl.sectoor.de",
+	// "residential.block.transip.nl",
+	"truncate.gbudb.net",
+	// "map.spam-rbl.com",
+	"rbl.schulte.org",
+	"xbl.spamhaus.org",
+	"spamguard.leadmon.net",
+	"noptr.spamrats.com",
+	// "forbidden.icm.edu.pl",
+	// "opm.tornevall.org",
+	// "ohps.dnsbl.net.au",
+	// "dnsbl.webequipped.com",
+	// "rbl.snark.net",
+	"dnsbl.inps.de",
+	"spam.pedantic.org",
+	// "sorbs.dnsbl.net.au",
+	"blacklist.woody.ch",
+	// "t3direct.dnsbl.net.au",
+	// "ubl.lashback.com",
+	// "ricn.dnsbl.net.au",
+	"dnsbl.kempt.net",
+	"dynip.rothen.com",
+	// "dsn.rfc-ignorant.org",
+	// "tor.dan.me.uk",
+	// "osps.dnsbl.net.au",
+	// "rbl.triumf.ca",
+	"web.dnsbl.sorbs.net",
+	// "socks.opm.blitzed.org",
+	// "l1.spews.dnsbl.sorbs.net",
+	// "rbl.spamlab.com",
+	"bogons.cymru.com",
+	"dyna.spamrats.com",
+	// "rbl-plus.mail-abuse.org",
+	// "dnsbl.cyberlogic.net",
+	"all.s5h.net",
+}
+
+// Reverse reverses slice of string elements.
+func Reverse(original []string) {
```

Comment:
Right. When I wrote this function, I decided to skip copying the whole slice. Then, writing `ReverseAddress`, I found I needed to get a new variable to be returned, so decided to make it different.
However, I think it would be better to have the same style in both functions. I added #16 to have this done later.